components/NewReplySection/ReplyForm/ReasonEditor.js

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import HelpIcon from '@material-ui/icons/Help';
+import CloseIcon from '@material-ui/icons/Close';
+import { Box } from '@material-ui/core';
+import { TYPE_SUGGESTION_OPTIONS } from 'constants/replyType';
+import { EDITOR_FACEBOOK_GROUP, EDITOR_REFERENCE } from 'constants/urls';
+
+const useStyles = makeStyles(theme => ({
+  editor: {
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    [theme.breakpoints.up('md')]: {
+      border: `1px solid ${theme.palette.secondary[100]}`,
+      borderRadius: 8,
+      '& > *:first-child': {
+        borderTopLeftRadius: 8,
+        borderTopRightRadius: 8,
+      },
+    },
+  },
+  tools: {
+    background: theme.palette.secondary[50],
+    [theme.breakpoints.up('md')]: {
+      background: theme.palette.common.white,
+      order: 1,
+    },
+    '& > *:not(first-child)': {
+      borderLeft: `1px solid ${theme.palette.secondary[100]}`,
+    },
+  },
+  suggestions: {
+    background: theme.palette.common.white,
+    color: theme.palette.secondary[300],
+    padding: 12,
+    whiteSpace: 'nowrap',
+    overflow: 'scroll',
```

Comment:
If what we want is a horizontal scroll when there is not enough space, please use `overflowX: auto`.

`overflow: scroll`:
- introduces always-on scrolls, it's there even if no scroll is needed
- scrolls both directions, which is pretty rare in design

![image](https://user-images.githubusercontent.com/108608/82548608-28c6cd00-9b8e-11ea-8d28-c12bcd4ee7a6.png)

I suggest web developers on Mac should turn on "always show scroll bars" at all times so that we can empathize Windows users more. It may be not obvious for Mac users with hidden scrollbars, the [Windows users will always get ugly scroll bars for every `overflow: scroll` we write.](https://www.filamentgroup.com/lab/scrollbars/)